src/top-nav/top-nav.component.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
```

Comment:
As mentioned above, this top-nav is going to be deprecated in favor of the hamburger menu nav in global-nav. We are working under the expectation that the items in that left-nav bar will be populated by the controlling app (in this case, patient-chart).